src/data-dictionary/events/ContainerSample/memoryUsageLimitPercent.md

```diff
@@ -0,0 +1,11 @@
+---
+name: memoryUsageLimitPercent
+type: attribute
+units: percentage (%)
+events:
+  - ContainerSample
+---
+
+This metric is calculated as the `memoryUsageBytes` percentage over `memorySizeLimitBytes`, if the limit exist.
+
+**Breaking change**: This metric is calculated using the total memory (including swap) over `memorySizeLimitBytes` on versions v1.5.0 or higher of `nri-docker`.
```

Comment:
Since this was actually a bug fix (percentage was not being correctly calculated) I would not note it here, just in the changelog.